necromancer.sh

```diff
@@ -8,7 +8,8 @@ oldApplicationCapitalizedName="Skeleton"
 oldDirectoryName="io/intrepid/skeleton"
 oldDirectoryPrefix="io/intrepid/"
 
-packageRegex="([a-z]+\.){1,3}([a-z]+)$"
+# We do not check for Java keywords... We should... but we don't. http://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
+packageRegex='^([a-z_][a-z_0-9]*\.)*([A-z_][A-z_0-9]+)$'
```

Comment:
@f2rf2r 
I wrote it to spec, well as close as i could... Truth be told case sensitivity in Bash for Regex relies on an environment variable... I could set that at the top of the script... but unless i export it the subshells created by `-exec` wont follow the rule but rather rely on what is currently set in the users login profile script. If i export it i have to play the 'save old', 'set 
 and export new', 'do stuff', 'reset and export' game... like i did with `LANG`. Note, my regex also only tolerates it at the last part of the name... I don't know why but in a lot of projects that seems to be an (unwritten/defacto) exception to the casing rule.

Let me know what your final decision is:

1. Leave it.
2. Just Change the Regex.
3. Enforce lowercase by ensuring the environment complies.